### PR TITLE
Kaspersky deletes Ignis upon download

### PR DESCRIPTION
**Pull request by zsannenmariano**
### Ticket
- [Kaspersky deletes Ignis upon download](https://trello.com/c/W88bo3Tc)

### Summary of Changes
- Updated "others" instruction
  - Included Kapersky as specified AV that does not work.

### CODE REVIEW

File: mining_tool_instruction/others/en.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Freedom! tries to work with many different antivirus companies to whitelist the official Crypto miner software. 
-Sometimes we still get a false positive and the antivirus software will block the installation or normal operation of the program. 
+Sometimes we still get a false positive and the antivirus software will block the installation or normal operation of the program. (Take note: That Kaspersky is one of the know antivirus that doesn't work)
```
Comment (jonelrempis) [inline mining_tool_instruction/others/en.html]: "known antivirus"

**Review by pandafox — APPROVED** [review]

File: mining_tool_instruction/others/en.html (hunk 1 above)
Comment (zsannenmariano) [reply mining_tool_instruction/others/en.html]: Fixed. Thanks for noticing. 😄

**Review by jonelrempis — APPROVED** [review]

**Review by cedricabuso — APPROVED** [review]

**Review by wildtron — APPROVED** [review]

**Review by wilbertverayin — APPROVED** [review]

**Review by ferdrian-melarpis — APPROVED** [review]

**Comment by kiko-f** [comment]
@zsannenmariano The note for Kaspersky exemption did not display in the Others tab

image

**Pull request merged**